examples/token/vipercoin.v.py

```diff
@@ -0,0 +1,95 @@
+# Viper Port of MyToken
```

Comment:
Can you change the file path to `examples/token/vipercoin.v.py` as I have a feeling we'll have multiple tokens examples down the road.

Reply:
Wait you mean `/tokens` right ? ;)

Reply:
Exactly :)